astrosat_users/serializers/serializers_auth.py

```diff
@@ -48,26 +44,19 @@ class LoginSerializer(ConsolidatedErrorsSerializerMixin, RestAuthLoginSerializer
     def validate(self, attrs):
         """
         Does the usual login validation, but as w/ LoginForm it adds some explicit checks for astrosat_users-specific stuff
-        (this is the right place to put it; the KnoxLoginView has its own KnoxTokenSerializer for users _and_ tokens,
-        but it uses this serializer for the user and so validation will be checked when processing the view.)
+        (this is the right place to put it; the KnoxLoginView has its own KnoxTokenSerializer for users and tokens,
+        but that uses this serializer for the user and so validation will be checked when processing the view.)
         """
-
-        instance = super().validate(attrs)
-
-        user = instance["user"]
-        user_serializer = UserSerializerLite(user)
+        self.instance = super().validate(attrs)
```

Comment:
why does this need to be an instance variable as opposed to a local variable, as it was before, I don't see it being referenced in other methods for instance?

Reply:
b/c I reference it in `LoginView.get_error_response`

Reply:
Sorry, should have seen that myself